errors/abci.go

```diff
@@ -111,19 +110,3 @@ func errIsNil(err error) bool {
 	}
 	return false
 }
-
-var errPanicWithMsg = Wrapf(ErrPanic, "error message redacted to hide potential sensitive info. Use the '--trace' flag if you are running a node to see the full stack trace error")
```

Comment:
are there some errors this should be the case? Do we know what this was meant to protect?

Reply:
the raw stacktrace.

Reply:
does this still need work then? I wasn't was going to mark r4r but if there is still info that needs to be hidden I can work on a solution

Reply:
No, nothing AFAIK needs to be hidden. Any thoughts here @amaurym?

Reply:
I'm resolving this :p